api/pom.xml

```diff
@@ -33,4 +33,21 @@
         <jjwt.root>${basedir}/..</jjwt.root>
     </properties>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.github.siom79.japicmp</groupId>
+                <artifactId>japicmp-maven-plugin</artifactId>
```

Comment:
How does this work exactly?  I found https://siom79.github.io/japicmp/ but it's unclear to me what we're using this plugin for.  Does it enforce semantic versioning?

Reply:
Yup, I've been using it in other projects for Semver (but we are still pre 1.0, so it will not block anything).  I'll add a comment to describe it a bit more
Right now it's just configured to break on binary breaking changes (source breaking change are allowed, e.g. adding methods to interfaces)